boards/arm/blt_plane/blt_plane.dts

```diff
@@ -0,0 +1,300 @@
+        zephyr,gain = "ADC_GAIN_1";
+        zephyr,reference = "ADC_REF_INTERNAL";
+        zephyr,acquisition-time = <ADC_ACQ_TIME_DEFAULT>;
+        zephyr,resolution = <12>;
+    };
+};
+*/
+&i2c1 {
+	pinctrl-0 = <&i2c1_scl_pb6 &i2c1_sda_pb7>;
+	pinctrl-names = "default";
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_FAST>;
+
+	ina0: ina219_battery@40 {
+		status = "okay";
+		compatible = "ti,ina219";
+		reg = <0x40>;
+		brng = <0>; // Set to 0 since we don't expect >16V
+		pg = <0>; // 7.5 mV drop.
+		sadc = <15>; // Shunt ADC: 12 bit - 128 sample averaging - 68.10 ms
+		badc = <15>; // Bus ADC: 12 bit - 128 sample averaging - 68.10 ms
+		shunt-milliohm = <5>;
+		lsb-microamp = <400>;
+	};
+
+	ina1: ina219_3v3@44 {
+		status = "okay";
+		compatible = "ti,ina219";
+		reg = <0x44>;
+		brng = <0>; // Set to 0 since we don't expect >16V
+		pg = <0>; // 0 mV drop
+		sadc = <15>; // Shunt ADC: 12 bit - 128 sample averaging - 68.10 ms
+		badc = <15>; // Bus ADC: 12 bit - 128 sample averaging - 68.10 ms
+		shunt-milliohm = <5>;
+		lsb-microamp = <33>;
+	};
+
+	ina2: ina219_5v0@41 {
+		status = "okay";
+		compatible = "ti,ina219";
+		reg = <0x41>;
+		brng = <0>; // Set to 0 since we don't expect >16V
+		pg = <0>; // 0 mV drop
+		sadc = <15>; // Shunt ADC: 12 bit - 128 sample averaging - 68.10 ms
+		badc = <15>; // Bus ADC: 12 bit - 128 sample averaging - 68.10 ms
+		shunt-milliohm = <5>;
+		lsb-microamp = <33>;
+	};
+
+	rv8263: rv8263@51 {
+		compatible = "microcrystal,rv-8263-c8";
+		reg = <0x51>;
+		status = "okay";
+		clkout = <1>;
+	};
+
+};
+
+&spi1 {
+	pinctrl-0 = <&spi1_sck_pa5 &spi1_miso_pa6 &spi1_mosi_pa7>;
+	cs-gpios = <&gpioa 8 GPIO_ACTIVE_LOW>;
+	pinctrl-names = "default";
+	status = "okay";
+
+	w5500: w5500@0 {
+		compatible = "wiznet,w5500";
+		reg = <0x0>;
+		spi-max-frequency = <DT_FREQ_M(80)>;
+		int-gpios = <&gpioa 9 GPIO_ACTIVE_LOW>;
+		reset-gpios = <&gpioa 10 GPIO_ACTIVE_LOW>;
+	};
+};
+
+&uart5 {
+	pinctrl-0 = <&uart5_tx_pc12 &uart5_rx_pd2>;
+	pinctrl-names = "default";
+	current-speed = <115200>;
+	status = "okay";
+};
+
+&dma1 {
+	status = "okay";
+};
+
+&dma2 {
+	status = "okay";
+};
+
+&spi2 {
+	pinctrl-0 = <&spi2_sck_pb10 &spi2_miso_pc2 &spi2_mosi_pc1>;
+	cs-gpios = <&gpioc 11 GPIO_ACTIVE_LOW>;
+	pinctrl-names = "default";
+	status = "okay";
+
+	// dma identifier
+	// dma stream
+	// dma channel
+	// config flags
+	dmas = <&dma1 4 0
+			(STM32_DMA_PERIPH_TX | STM32_DMA_PRIORITY_HIGH | STM32_DMA_OFFSET_FIXED_4 | STM32_DMA_PERIPH_8BITS | STM32_DMA_MEM_8BITS)
+			0x1>,
+		   <&dma1 3 0
+			(STM32_DMA_PERIPH_RX | STM32_DMA_PRIORITY_HIGH | STM32_DMA_OFFSET_FIXED_4 | STM32_DMA_PERIPH_8BITS | STM32_DMA_MEM_8BITS)
+			0x1>;
+	dma-names = "tx", "rx";
+
+	w25q128: w25q128jv@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+
+		spi-max-frequency = <DT_FREQ_M(104)>;
+		size = <DT_SIZE_M(128)>;
+		jedec-id = [ef 40 18];
+
+		has-dpd;
+		t-enter-dpd = <(3 * 1000)>;
+		t-exit-dpd = <(3 * 1000)>;
+
+		partitions {
```

Comment:
same idea as comment from above about not putting lfs partition in board dts

i unwisely setup storage partitions in control_freak board dts and now have to do silly delete node dts magic in the airbrake program